src/pages/news/newsDetails/NewsDetail.module.css

```
@@ -0,0 +1,100 @@
+    width: 100%;
+    max-width: 100%; /* Limite stricte */
+    box-sizing: border-box;
+    overflow-x: hidden; /* Bloque le scroll horizontal interne */
+}
+
+.richContent {
+    width: 100% !important;
+    max-width: 100% !important;
+    overflow-wrap: break-word !important; /* Force la coupure des mots longs */
+    word-wrap: break-word !important;
+    word-break: break-word !important;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)